src/mappers/country/country_mapper_creator.cpp

```diff
@@ -33,7 +36,10 @@ std::map<std::string, std::string> ImportMappingRules(std::string_view country_m
           vic3_tag.clear();
           hoi4_tag.clear();
           mapping_rule_parser.parseStream(input_stream);
-          country_mapping_rules.emplace(vic3_tag, hoi4_tag);
+          if (!kDisallowedTags.contains(hoi4_tag))
+          {
+             country_mapping_rules.emplace(vic3_tag, hoi4_tag);
```

Comment:
Yknow, if you're gonna just ignore a semantically valid rule from the mapping file, at least throw out a warning that the rule is invalid due to this and that.